modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/source-builder/serializeUtil.js

```diff
@@ -172,8 +172,8 @@ function appendXMLAssignments(
 					);
 
 					if (
-						dataAssignments.autoCreate[index] !== null &&
-						dataAssignments.autoCreate[index] !== undefined
+						dataAssignments.autoCreate?.[index] !== null &&
```

Comment:
@alinedoleron you can just replace the entire condition with `dataAssignments.autoCreate?.[index]` :)